udata/commands/static.py

```diff
@@ -24,29 +24,45 @@
 def collect(path, input):
     '''Collect static files'''
     if exists(path):
-        print('"{0}" directory already exists and will be erased'.format(path))
+        msg = '"%s" directory already exists and will be erased'
+        log.warning(msg, path)
```

Comment:
Just curious, what are the pros of `msg, path` (vs. `msg % path` or `msg.format(path)`)?